gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -550,17 +551,21 @@ private LocalResourceManagerHelper() {
   }
 
   /**
-   * Creates a LocalResourceManagerHelper object that listens to requests on the local machine.
+   * Creates a {@code LocalResourceManagerHelper} object that listens to requests on the local
```

Comment:
Nit: links to self in javadoc are not useful, we could use `{@code LocalResourceManagerHelper}` instead.